src/js/tracks/text-track.js

```diff
@@ -19,7 +19,25 @@ import XHR from 'xhr';
  * @param {String} srcContent webVTT file contents
  * @param {Track} track track to addcues to
  */
-const parseCues = function(srcContent, track) {
+const parseCues = function(srcContent, track, retryNum) {
```

Comment:
should retryNum default to 0 here since we are not passing it in the first time?